pkg/daemon/ceph/osd/volume.go

```diff
@@ -262,17 +262,15 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 				devices.Entries["metadata"].Config.Name,
 			}...)
 
-			crushDeviceClass := os.Getenv(oposd.CrushDeviceClassVarName)
-			if crushDeviceClass != "" {
-				metadataArg = append(metadataArg, []string{crushDeviceClassFlag, crushDeviceClass}...)
-			}
 			metadataBlockPath = devices.Entries["metadata"].Config.Name
 		}
 
 		if device.Data == -1 {
 			logger.Infof("configuring new device %q", device.Config.Name)
 			var err error
 			var deviceArg string
+			var immediateExecuteArgs []string
```

Comment:
Why declare this var here? It's preferred to declare the variables near where they are used, so I would suggest not changing line 284.

Reply:
okay, changes are done.